doc/source/whatsnew/v0.20.0.txt

```diff
@@ -1,12 +1,16 @@
 .. _whatsnew_0200:
 
-v0.20.0 (????, 2016)
+v0.20.0 (????, 2017)
 --------------------
 
 This is a major release from 0.19 and includes a small number of API changes, several new features,
 enhancements, and performance improvements along with a large number of bug fixes. We recommend that all
 users upgrade to this version.
 
+.. warning::
+
+   Due to important fixes and performance improvements, ``numexpr`` version is now required to be >= 2.4.6 and it will not be used at all if this requisite is not fullfilled (:issue:`15213`).
```

Comment:
it's an API change for pandas meaning if u has a version that is no longer supported then things will change - its also something to be aware of when upgrading